app/web/markdown/blog/2024/09/08/summer-2024-updates.md

```diff
@@ -0,0 +1,98 @@
+
+There's also a few less visible or less exciting ones:
+
+* **We removed all password optinality in the backend**: there were some relics of the old password-optional regime that have been removed, and we also simplified the login form to show the username and password inputs straight away.
+* **A variety of behind-the-scenes upgrades**: we've also done a lot of work that's not very visible. There was a really gnarly bug that was causing havok for a week or so but that we managed to squash last Sunday. We've also created a bunch of monitoring and observability infrastructure that has helped us speed up some parts of the platform that, you might notice it's a bit more snappy now (but there's more to come)! We made some upgrades to the infrastructure that helps us run the project. Finally we also built some new admin tooling that helps mods keep the platform fun and friendly with much less effort!
+* **New support infrastrucutre**: we switched out our support platform from a very old and flaky open source platform to a much newer and better platform called Chatwoot! We also switched from using a commercial mailing list provider to a self-hosted platform that will save us a bunch of money.
+* **You now "decline" requests** instead of "rejecting" them, because this sounds friendlier and Couchers is a friendly place :).
+
+
+## Celebrating our community of 43k Couchers!
+
+Couchers is continuing to grow quickly with over 43,000 registered Couchers! Our most important goal right now is to get more couch surfers into our community and onto the platform.
+
+You can help us with this goal by sharing the platform and spreading the word, as well as continuing to use the platform, provide feedback, and helping us grow!
+
+
+## Our long-term strategic goal: the safest couch surfing platform
+
+For a long time our marketing has centered around being "like CouchSurfing™, but better". While this is still the core vision and has resonated strongly with "CouchSurfing™ refugees", we believe we now need to do some rethinking of our branding. We are now approaching a point where we need to appeal to fist time couch surfers and better explain what sets us apart from other platforms.
+
+We believe the core "currency" of couch surfing is the trust and safety we have in other humans. This is also (by no coincidence) the biggest inhibitor of outsiders jjoining our community. Through our many past, current, and future efforts we believe Couchers can effectively reposition ourselves to be the **safest couch surfing platform** out there. We believe this is one of the key factors missing from CouchSurfing.com™ and is simultaneously the primary concern of new users. By focussing our efforts and image around this goal, we think we can best serve both current and future couch surfers looking for the most genuine, safe platform for couch surfing.
```

Comment:
```suggestion
We believe a core value of couch surfing is the trust and safety we have in other humans. This is also (by no coincidence) the biggest inhibitor of outsiders joining our community. Through our many past, current, and future efforts, we believe Couchers can effectively position ourselves to be the **safest couch surfing platform** out there. By focussing our efforts around this goal, we think we can best serve both current and future couch surfers looking for the most safe platform for couch surfing.
```

Reply:
The word currency shouldnt be used in the context of couchsurfing since it is supposed to be non-transactional. We also can't say what users' "primary concerns" are or what "key factors" are on CS... many people on CS are happy with the safety, and CS does have a safety system. Its just shittier than what it COULD be. It USED to be a lot lot better. The main thing we want to drive home here is that we want to be MORE safe than CS.

Reply:
Hmmmm.... I do think "currency" is apt here, but I've gotten rid of it anyway :)